### PR TITLE
Inc-15/textarea

### PR DESCRIPTION
**Pull request by ilich01**
Create an textarea items for ui-kit and add it to the ui-kit page. 
image

### CODE REVIEW

File: pages/dev/ui-kit/index.tsx [hunk 1]
```diff
@@ -44,24 +55,55 @@ const Login = () => {
...
+              <TextArea color={TEXTAEREA_COLORS.DEFAULT} />
+            </form>
+            <form onSubmit={handleSubmit(onSubmit)}>
+              <TextArea color={TEXTAEREA_COLORS.ACTIVE} isActive={true} />
+            </form>
+            <form onSubmit={handleSubmit(onSubmit)}>
+              <TextArea color={TEXTAEREA_COLORS.ERROR} hasError={true} />
+            </form>
+            <form onSubmit={handleSubmit(onSubmit)}>
+              <TextArea color={TEXTAEREA_COLORS.HOVER} isHovered={true} />
```
Comment (TURP1) [inline pages/dev/ui-kit/index.tsx:71]: you need to throw this logic inside, classes like hover, active should be used in the component automatically, you don't need to write them manually.

File: @ui/ui-kit/Textareas/Textarea.tsx [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+  onBlur?: (event: FocusEvent<HTMLTextAreaElement>) => void
+}
+
+const TextArea: React.FC<TextAreaProps> = ({
+  defaultValue,
+  isActive = false,
+  hasError = false,
+  isHovered = false,
+  isFocused = false,
+  isDisabled = false,
```
Comment (TURP1) [inline @ui/ui-kit/Textareas/Textarea.tsx:27]: - isDisabled -> disabled 
- we dont need to have isHovered, isFocused etc... in props -  it`s pseudoclasses , use :hover :focus is SCSS

Comment (TURP1) [reply @ui/ui-kit/Textareas/Textarea.tsx:27]: tooooo much props for such little component , think about how difficult is using such Component

File: @ui/ui-kit/Textareas/Textarea.module.scss [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+      /* regular_text 16 */
+      font-family: Inter;
+      font-size: 16px;
+      font-style: normal;
+      font-weight: 400;
+      line-height: 24px; /* 150% */
+      margin: 16px 24px;
+    }
+  }
+  &Active {
```
Comment (TURP1) [inline @ui/ui-kit/Textareas/Textarea.module.scss:42]: :active

File: @ui/ui-kit/Textareas/Textarea.tsx [hunk 4]
```diff
@@ -0,0 +1,56 @@
+import React, { ChangeEvent, FocusEvent } from 'react'
+import classNames from 'classnames'
+import styles from './TextArea.module.scss'
```
Comment (Viarcinski) [inline @ui/ui-kit/Textareas/Textarea.tsx:3]: Your file is called "Textarea.module.scss" but you are importing it as "TextArea.module.scss". This throws an error on vercel

File: @ui/ui-kit/Textareas/Textarea.module.scss (hunk 3 above)
Comment (Viarcinski) [reply @ui/ui-kit/Textareas/Textarea.module.scss:42]: The same thing with hover, focus and disabled

**Review by Viarcinski — CHANGES_REQUESTED** [review]

**Comment by ilich01** [comment]
Refactoring: Slightly moved away from the layout, made separate properties for focus and active for color ERROR. properties: active, hover can be checked through the property inspector 
image

**Review by irinrudko — APPROVED** [review]

**Review by TURP1 — CHANGES_REQUESTED** [review]
:active isn`t woking onClick default textarea

**Review by maskpa100 — APPROVED** [review]

**Comment by TURP1** [comment]
Should be :

Done by you
Знімок екрана 2023-07-27 о 22 47 11

**Review by maskpa100 — APPROVED** [review]

**Review by TURP1 — APPROVED** [review]

**Review by Viarcinski — APPROVED** [review]

**Review by maskpa100 — APPROVED** [review]

**Pull request merged**